samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitor.java

```diff
@@ -147,6 +145,8 @@ private void markSweepTaskStore(File taskStoreDir) throws IOException {
     if (!offsetFile.exists()) {
       LOG.info("Deleting the task store : {}, since it has no offset file.", taskStorePath);
       FileUtils.deleteDirectory(taskStoreDir);
+      localStoreMonitorMetrics.diskSpaceFreedInBytes.inc(taskStoreDir.getTotalSpace());
```

Comment:
Does getTotalSpace() return an appropriate value AFTER the dir has already been deleted (line 147)?

How was this tested?